nimbus-features/shareExtensionCoordinatorRefactor.yaml

```diff
@@ -0,0 +1,19 @@
+# The configuration for the shareExtensionCoordinatorRefactor feature
+features:
+  share-extension-coordinator-refactor:
+    description: >
+      This feature is for managing the roll out of the share extension coordinator as part of the changes to the navigation in the app
+    variables:
+      enabled:
+        description: >
+          Enables the feature
+        type: Boolean
+        default: false
+    defaults:
+      - channel: beta
+        value:
+          enabled: false
+      - channel: developer
+        value:
+          enabled: true
```

Comment:
This should be disabled in the developer channel until the feature is complete.

Reply:
Ok perfect i modified it right now